lib/components/molecules/Popover/types/index.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  popOverBody: Node | string,
+  popOverHeader: Node | string,
+  children: React$Element<*>,
```

Comment:
For Children correct type is Node

Reply:
Initially I defined children as node (in new use case - its trigger prop) but getting Linting error. On github flow 
 issue thread I found that if an element is cloned then it should be mentioned as React$Element<*>